### DISCUSSION TITLE
The "Most used languages" bar doesn't work correctly

### DISCUSSION THREAD

**Discussion by forgedhallpass** [lowlighter/metrics#1202] [Q&A]
This is the generated snippet for my profile, but I have barely touched Ruby code. My top contributions are in Java and GoLang, but that's not reflected correctly by far.

![image](https://user-images.githubusercontent.com/13679401/186644985-5201d25c-bac3-4c08-96ea-a18ee79d1358.png)

**Comment by spenserblack** [comment]
This is likely because of [your homebrew fork](https://github.com/forgedhallpass/homebrew-core). The [`plugin_languages_indepth`](https://github.com/lowlighter/metrics/blob/master/source/plugins/languages/README.md#%EF%B8%8F-available-options) can help with this, as it analyzes your commits to see what you've *actually* written. You can also try unsetting `repositories_forks`.

**Reply by forgedhallpass** [reply]
Thanks for the tips, but if I disable the `repositories_forks`, then for some reason it isn't able to extract my `go` contributions (even though they were not done in forks):
![image](https://user-images.githubusercontent.com/13679401/188490031-fe60c95d-8424-4634-b64d-f9eb836e9ffa.png)

If I enable `plugin_languages_indepth`, then it's slightly better, but still it doesn't show the right values, since I have more `go` contributions than `JavaScript`:

![image](https://user-images.githubusercontent.com/13679401/188489915-82edd06e-62e2-4798-ac95-46cb32b6f14e.png)

**Reply by spenserblack** [reply]
[According to your GitHub profile](https://github.com/forgedhallpass?tab=repositories&q=&type=&language=go&sort=) your 2 Go repositories are forks. You might have private Go projects that metrics isn't detecting.

**Reply by forgedhallpass** [reply]
I have public `go` contributions in multiple repositories within the ProjectDiscovery organization.

For example: https://github.com/projectdiscovery/nuclei/graphs/contributors
